.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+# Description
+
+Please include a summary of the change and which issue is fixed (if relevant).
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How to test this change?
+
+Please describe the steps to test your changes.
+
+- Step 1
```

Comment:
Let's clarify this a little bit more. What are we expecting? A HCL file? Set of commands?